organizing_hub/templates/calls/callcampaign_form.html

```diff
@@ -33,9 +33,12 @@
 
     <div class="col-md-9">
 
-      <form action="{% url 'organizing-hub-call-campaign-create' %}"
-      method="post" class="form mb20">
-
+      {% if update_view %}
+      {% url 'organizing-hub-call-campaign-update' object.uuid as action_url %}
```

Comment:
Suggest to be consistent, either use `campaign ` or `object` in the template. I like the more semantic version, rather than `object`.